wiki/Tournaments/UCUP/2021/en.md

```diff
@@ -0,0 +1,559 @@
+| Upper | **Surtr S3** | **5** | 0 | Kotori | [#1](https://osu.ppy.sh/community/matches/91456494) |
+| Upper | **L9 Hamster** | **5** | 0 | dead rats | [#1](https://osu.ppy.sh/community/matches/91459292) |
+| Upper | **yo** | **5** | 1 | whatdoiputhere | [#1](https://osu.ppy.sh/community/matches/91462221) |
+| Upper | **momoteaStans** | **5** | 1 | yoinkysploinky | [#1](https://osu.ppy.sh/community/matches/91465940) |
+| Upper | :hinamizada: | 0 | **5** | **:markus:** | *win by default* |
+
+Sunday, 26 September 2021:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Upper | **Venception** | **5** | 0 | Miko Bloom | [#1](https://osu.ppy.sh/community/matches/91505178) |
+| Upper | **CrezzMyMommy** | **5** | 0 | marcusfancam | [#1](https://osu.ppy.sh/community/matches/91513760) |
+| Upper | **having fun** | **5** | 1 | osu winx club | [#1](https://osu.ppy.sh/community/matches/91502478) |
+| Upper | **i love tateyama** | **5** | 1 | skor buruk | [#1](https://osu.ppy.sh/community/matches/91503387) |
+| Upper | **:iofiPanic:** | **5** | 0 | Pegmatite | [#1](https://osu.ppy.sh/community/matches/91517616) |
+| Upper | **oogey rrrrat** | **5** | 0 | taketake\_gif | [#1](https://osu.ppy.sh/community/matches/91521776) |
+| Upper | **Spacediver** | **5** | 0 | idk any | *win by default* |
+
+## Ruleset
+
+### General rules
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+2. The mappools for each round will be announced by the Tournament Management in advance before the actual matches take place.
+3. Match schedules will be predetermined by the Tournament Management. If there are any player(s) or team(s) who are unable to attend the current schedule for any reason, all other affected teams may apply and settle for a reschedule at the `#reschedule` channel in the tournament's Discord server.
+4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+5. If a team does not show up within **10 minutes** of the start time, their opponent wins by default.
+6. If no staff or referee is available, the match will be postponed.
+7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards players and teams who perform better in general during the course of the beatmap regardless of their remaining health at the end.
+8. If a player disconnects, it will be treated as if they had failed the beatmap. 
+   - A match can be rematched for disconnects that occur within a few seconds after the beatmap has been started by the referee.
+9. If a player disconnects between beatmaps, the match can be delayed up to 15 minutes at most. 
+   - In case the disconnected player fails to report back to their team after the maximum allowance of 15 minutes had passed, the opposing team may be declared to win the match by default.
```

Comment:
```suggestion
   - In case the disconnected player fails to report back to their team after the maximum allowance of 15 minutes pass, the opposing team may be declared to win the match by default.
```